test/property.cpp

```diff
@@ -15,6 +15,8 @@
 
 BOOST_AUTO_TEST_SUITE(test_property)
 
+using namespace mqtt::literals;
```

Comment:
Why not define the mqtt::literals namespace as an "inline" namespace, so that you don't need to have these using statements everywhere?

Reply:
I didn't come up with that. I will fix it.

Reply:
I agree with add `inline` for namespace `literals`.
BTW, even if I add it, it seems that user code doesn't omit `using namespace mqtt::literals`. At least user need to write `using namespace mqtt`. But it is not good. If I write the code using `mqtt_cpp`, I use `mqtt::some` for use `mqtt_cpp` features and `using namespace mqtt::literals` for ""_mb.

Am I missing something?

Reply:
I mean https://wandbox.org/permlink/MH4AZ2FFp10TYSvQ

Reply:
User code would still need to do:

`using namespace mqtt::literals`

but it allows your code to avoid needing to use it.

Reply:
Right. 
BTW, there is no place using ""_mb in my library code, so far. That is the reason I got confused.

Reply:
Oh. Ok, I didn't realize that.

Thanks for clarifying.